BILLthebuilder/automart
# #166809341 Create endpoint to delete a specific car ad

* It creates an endpoint for a user to delete a specific car ad
#### Description of Task to be completed?
* The `DELETE /api/v2/car/:id` endpoint is created here
#### How should this be manually tested?
* Clone the repository
* Checkout to the `ft-delete-specific-166809341` branch
* Clone the repo
* Run `npm install` to install package dependencies
* Run `npm test` to run tests
* All the tests should pass
* Run `npm start`
* Run the DELETE `api/v1/car/:id` to delete a specific ad
* It should return a `status` of `200` with a telling you the ad was deleted
* It should return a `status` of `404` if the ad record is non-existent
* If you run the GET `api/v2/cars/` to view all posted ads, the deleted ad will be missing
#### What are the relevant pivotal tracker stories?
* [#166809341](https://www.pivotaltracker.com/story/show/166809341)
#### Screenshots
![Screenshot from 2019-06-20 06-05-56](https://user-images.githubusercontent.com/24655101/59815844-92a57980-9322-11e9-98cf-51080501d302.png)
